### DISCUSSION TITLE
Loading multiple datasets in pycaret

### DISCUSSION THREAD

**Discussion by QueSabz** [pycaret/pycaret#2355] [Q&A]
Is it possible to load multiple datasets in pycaret? 
for example lets say a list like this Data_arr = ['Poker','Covtype','Susy','Spam','Higgs','Letter','Biodegrable','Bank','HTRU','Magic'] with different number of dimensions

**Comment by ngupta23** [comment]
Assuming these are inbuilt datasets, you could do something like this

```python
datasets = ['Poker','Covtype','Susy','Spam','Higgs','Letter','Biodegrable','Bank','HTRU','Magic']
all_data = [get_data(dataset) for dataset in datasets]
```

**Reply by QueSabz** [reply]
These are external datasets....Could it be still possible to do the same?

**Reply by ngupta23** [reply]
I am not sure what you are trying to do with the multiple datasets. Pycaret only works on 1 dataset at a time, but you can externally loop your pycaret logic over multiple datasets (this is independent of pycaret and more python specific).
